front/src/RefboxField/index.js

```diff
@@ -44,7 +44,11 @@ const RefboxField = props => {
               id={`${tag}_magenta`}
               onClick={() => {
                 console.log(`${TAG_VALUES[tag]}_magenta`);
-                sendToServer(`${TAG_VALUES[tag]}_magenta`, "command");
+                sendToServer(
+                  `${TAG_VALUES[tag]}_magenta`,
+                  "command",
+                  props.token
```

Comment:
Please add "token" to the props in the documentation

Reply:
done